### PR TITLE
Task/mobile 1414/update native SDK to v1.6.1

### PR DESCRIPTION
**Pull request by AlonShprung**
#### Main purpose:
(Why? "User Story"/Bug description - what's in for the user/SDK client/this repo developer)
#### Ticket:
https://spotim-jira.atlassian.net/browse/MOBILE-1414
#### Technical description:
(What was changed/added)
#### Notes:
(Important stuff that reviewer should notice/ "breaking changes?") (Not a must)

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -322,7 +322,7 @@ DEPENDENCIES:
   - React-jsi (from `../node_modules/react-native/ReactCommon/jsi`)
   - React-jsiexecutor (from `../node_modules/react-native/ReactCommon/jsiexecutor`)
   - React-jsinspector (from `../node_modules/react-native/ReactCommon/jsinspector`)
-  - "react-native-spotim (from `../node_modules/@spot.im/react-native-spotim`)"
+  - react-native-spotim (from `../..`)
```
Comment (AlonCherrypick) [inline example/ios/Podfile.lock:325]: @oded-regev I'm not sure why it was changed. Any idea?

File: example/ios/Podfile.lock [hunk 2]
```diff
@@ -380,7 +380,7 @@ EXTERNAL SOURCES:
   React-jsinspector:
     :path: "../node_modules/react-native/ReactCommon/jsinspector"
   react-native-spotim:
-    :path: "../node_modules/@spot.im/react-native-spotim"
+    :path: "../.."
```
Comment (AlonCherrypick) [inline example/ios/Podfile.lock:383]: @oded-regev I'm not sure why it was changed. Any idea?

File: example/ios/example/GoogleAdsProvider.swift [hunk 3]
```diff
@@ -83,48 +82,39 @@ import SpotImCore
             return kGADAdSizeMediumRectangle
         }
     }
+  
+    @objc public class func setSpotImSDKWithProvider() {
+      SpotIm.setGoogleAdsProvider(googleAdsProvider: GoogleAdsProvider())
+    }
```
Comment (AlonCherrypick) [inline example/ios/example/GoogleAdsProvider.swift:88]: @oded-regev This code was added to GoogleAdsProvider file only in RN. We should update the files in the pod repo.

**Review by oded-regev — APPROVED** [review]
It is generally approved, just please make sure to fix the parts we talked about just now

**Pull request merged**